### PR TITLE
updating link to code of conduct in org repository

### PR DESCRIPTION
**Pull request by vsoch**
Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

Here is 3/3, updating the code of conduct link.

### CODE REVIEW

**Comment by vbatts** [comment]
LGTM

**Comment by cyphar** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The Go types and validation should be compatible with the current Go release; ea
 
 Additional documentation about how this group operates:
 
-- [Code of Conduct](https://github.com/opencontainers/tob/blob/d2f9d68c1332870e40693fe077d311e0742bc73d/code-of-conduct.md)
+- [Code of Conduct](code-of-conduct)
```
Comment (cyphar) [inline README.md]: Ah, I missed that this isn't actually a correct URL -- it should be `[code-of-conduct]`.

Comment (vsoch) [reply README.md]: Ack, my mistake I'll fix this.

**Pull request merged**